integration_tests/src/main/python/iceberg_test.py

```diff
@@ -37,6 +37,7 @@
 
 rapids_reader_types = ['PERFILE', 'MULTITHREADED', 'COALESCING']
 
+pytestmark = pytest.mark.skip(reason="Skipping all iceberg tests as it's under refactoring")
```

Comment:
Let us refer to the epic or some dedicated issue in the reason. I also prefer an xfail since it should eventually work and we can track XPASS
```suggestion
pytestmark = pytest.mark.xfail(reason="Skipping all iceberg tests as it's under refactoring https://github.com/NVIDIA/spark-rapids/issues/12176")
```